example/edb.yml

```diff
@@ -7,6 +7,10 @@
     :database: sample_database
     :host: localhost
     :port: 5432
+    :only_tables: ['table1', 'table2']
+    :without_tables: ['table1', 'table2']
```

Comment:
I suggest to rename this option to `exclude_tables` to match the name used by PostgreSQL.